ci/pipeline

```diff
@@ -368,5 +368,5 @@ def release(args: String*): Unit = {
       %('sbt, "plugin-interface/publish")
   }
 
-  githubClient.tag(config.version.commit, config.version.toTagString)
+  // githubClient.tag(config.version.commit, config.version.toTagString)
```

Comment:
can we just delete this line? I never want the release job doing tags. It's too weird.

Reply:
What is the motivation for removing this line?

Reply:
1. the CI job doesn't have authorization to tag
2. there is less of a reason to tag something that has a nature tag (the rev is in the build)
3. we have traditional manually affix a release tag which will require human involvement to provide release not details.

Reply:
Ah, ok, I thought each community release has to have a tag. Feel free to remove this line.